### ISSUE TITLE
Image after uploading is not visible and may be hard to find 

### ISSUE THREAD

**Issue by tkacheva** [magento/adobe-stock-integration#1191]
## Issue 
When file is uploaded, it's not in the viewport because images in the gallery are sorted the way, file is added to X page. So it looks like the file is not uploaded at all. 

## Preconditions
Images in Media Gallery grid sorted by  name, directory or oldest images first 

## Steps 

- Open media gallery
- Drag images to the media gallery grid or Upload images with file uploader
- ❌ When images processed observe that grid did't change and uploaded images are not visible 
- ✅ Confirmation message displayed with the option for user to view just uploaded images

## Design
![Image uploaded - notification](https://user-images.githubusercontent.com/752610/79154806-d2e45580-7d95-11ea-876e-55755e40ca90.png)

Basically, have a notification "Successfully uploaded X images" or "Successfully uploaded an image" if only one image.

Additionally, add a button "Show newest" if sort order in gallery is not by "Newest first".

**Comment by belbiy** [comment]
Please see the proposed solution:
![Image uploaded - notification](https://user-images.githubusercontent.com/752610/79154806-d2e45580-7d95-11ea-876e-55755e40ca90.png)


Basically, have a notification "Successfully uploaded X images" or "Successfully uploaded an image" if only one image.

Additionally, add a button "Show newest" if sort order in gallery is not by "Newest first".

**Comment by sivaschenko** [comment]
Currently when images are uploaded - the newest first sorting is applied. @tkacheva is this issue still actual?
